homeassistant/components/onvif/camera.py

```diff
@@ -256,6 +256,7 @@ def __init__(self, hass, config):
 
             _LOGGER.debug("Retrieving stream uri")
 
+            media_service = self._camera.create_media_service()
```

Comment:
# Fix Onvif setup error on cheap chinese cam: premature end of connection on GetStreamURI #26781